### ISSUE TITLE
URL var is declared multiple times in lib files

### ISSUE THREAD

**Issue by myitcv** [Microsoft/TypeScript#5306]
```
$ grep -n 'declare var URL' lib/*
lib/lib.dom.d.ts:10903:declare var URL: URL;
lib/lib.dom.d.ts:12945:declare var URL: URL;
lib/lib.d.ts:14727:declare var URL: URL;
lib/lib.d.ts:16769:declare var URL: URL;
lib/lib.es6.d.ts:16042:declare var URL: URL;
lib/lib.es6.d.ts:18084:declare var URL: URL;
```

Just saw that #5293 was merged; happy to submit a PR unless these files are generated in some way?

**Comment by mhegazy** [comment]
@zhengbli can you take a look.

**Comment by mhegazy** [comment]
PRs are welcomed. here is some details about submitting changes to lib.d.ts: https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes

**Comment by myitcv** [comment]
This is fixed as of `1.9.0-dev.20160408`:

```bash
$ npm install typescript@1.9.0-dev.20160408
...
$ grep -n 'declare var URL' node_modules/typescript/lib/* | grep dom\.d\.ts
node_modules/typescript/lib/lib.dom.d.ts:11765:declare var URL: {
```

@zhengbli the only outstanding issue then is https://github.com/Microsoft/TypeScript/pull/7619#issuecomment-207550804. I'll create an issue to track that
